src/pages/workspaces/workspace/analysis/tool-utils.ts

```diff
@@ -32,6 +34,7 @@ export interface Tool {
   isHidden?: boolean
   isLaunchUnsupported?: boolean
   isPauseUnsupported?: boolean
+  isSettingsUnsupported?: boolean
```

Comment:
How is `isSettingsUnsupported` different from `isHidden`?

Reply:
I though it would be cleaner to keep it separate as well as in future we would not need to keep the app hidden but we would still require the settings option to be disabled as the app will be created when the workspace is created.